### ISSUE TITLE
 Warning: Expo version in package.json does not match sdkVersion in manifest.

### ISSUE THREAD

**Issue by ryanhomer** [expo/expo#793]
Hello,

I'm new to Expo and taking over a project. I [followed the instructions](https://blog.expo.io/expo-sdk-21-0-0-is-now-available-be33b79921b7) to upgrade to SDK 21 and Expo now tells me:
```
Warning: Expo version in package.json does not match sdkVersion in manifest.
```

App.json:
```
{
  "expo": {
    ...
    "sdkVersion": "21.0.0",
    "version": "1.0.0",
    ...
  }
}
```

package.json:
```
  "dependencies": {
    "react": "16.0.0-alpha.12",
    "react-native": "https://github.com/expo/react-native/archive/sdk-21.0.2.tar.gz",
    ...
  },
  "devDependencies": {
    "jest-expo": "^21.0.0",
    ...
  }
```

I've nuked `node_modules` and done `npm install` again to no avail. Is there anywhere else I'm supposed to check for an SDK version mismatch?

**Comment by ide** [comment]
This looks right. You can probably ignore that warning if everything else seems to be working.

**Comment by ryanhomer** [comment]
Just getting back to my project today, started up Expo and am not getting the warning anymore. Caching issue? We may never know. 🙁

**Comment by Plorax** [comment]
So, Happy New Year Everyone !

Firstly, SDK version mismatch is between your App.json file, that contains the sdkVersion, matched to the actual Expo client you're running on your mobile. Mine happened to be 23.0.0 in App.json and 24.0.0 in the client app. So I updated my App.json to 24.0.0 and that warning went away.

I hope this helps anyone out there. I've searched for hours to find a solution. I hate warnings so this was a huge help.


Ciao.
